Makefile

```diff
@@ -43,19 +43,17 @@ quicktest:
 test: appfortests
 	docker-compose run --rm test
 
-testcli: appfortests tablesfortests externalapi
+testcli: appfortests tablesfortests mfaapi
 	docker-compose run --rm test bash
 
-externalapi:
-	docker-compose up -d external_api
+mfaapi:
+	docker-compose up -d mfaapi
 
 clean:
 	docker-compose kill
 	docker system prune -f
 
-raml2html: api.html
-
-api.html: api.raml
+raml2html:
```

Comment:
It was done this way so Make could skip the build if no changes were found in the raml file. Pretty insignificant difference, so the change is fine.

Reply:
It wasn't building for me even though the api.html file was empty.